### ISSUE TITLE
Waaaaay down the road feature request:  Flyer-style compatibility

### ISSUE THREAD

**Issue by xlar54** [pi1541/Pi1541#80]
There was once a device called the Flyer (http://www.retroswitch.com/products/flyer/#) which provided 1541/71/81 drive capability AND ethernet access through IEC.  Sounds like writing to device #12 or something would send commands to and from the device to talk to an ethernet tcp stack on the device.  Since its just IEC commands, this could be implemented in your code as well.  Apparently this method saves the 64 from needing its own tcp stack in memory, taking up a ton of RAM.  Dunno if you label tickets, but maybe you could label this as a feature request.  Would make an excellent wifi ethernet device and better than the rs232 / tcp wifi modem emulators.

**Comment by idolpx** [comment]
I don't think it gives you access to the ethernet connection. It uses the connection to mount disk images over the network though and feed them to the 64 like it is a floppy drive.
